src/Console/CacheEventHandlersCommand.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Spatie\EventProjector\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+use Illuminate\Filesystem\Filesystem;
+use Spatie\EventProjector\Projectionist;
+use Spatie\EventProjector\EventHandlers\EventHandler;
+
+final class CacheEventHandlersCommand extends Command
```

Comment:
Is this command necessary? Can't we just always cache in production? I don't really want to clutter my deployment script with every package that needs to cache itself.

Reply:
I think it's good to have a bit of control over the caching, I don't mind adding stuff like this to the deployment script